### ISSUE TITLE
Suggestion. Add RSS

### ISSUE THREAD

**Issue by Daniel-at-github** [koaning/calmcode-feedback#96]
New content is announced via a newsletter.
Mi suggestion is to add RSS as an alternative.

**Comment by koaning** [comment]
Do you have a good resource on how to make a proper RSS feed?

**Comment by Daniel-at-github** [comment]
It's a surprisingly non trivial question.

Looking into what known projects are using, that generates rss, my two main options are:
* [python-feedgen](https://github.com/lkiesow/python-feedgen) is popular and [documented](https://feedgen.kiesow.be/), but [don't have activity](https://github.com/lkiesow/python-feedgen/issues/106) since the beginning of 2020.
* [feedgenerator](https://github.com/getpelican/feedgenerator) with recent activity and used by 2.7k projects, but with minimal [documentation](https://github.com/getpelican/feedgenerator/blob/master/tests_feedgenerator/usage_example.py)  (or you have to extrapolate from [django rss docs](https://docs.djangoproject.com/en/3.2/ref/contrib/syndication/))

**Comment by koaning** [comment]
This is still on my todo, but yeah, since it's a non-trivial thing to get right I might delay the implementation.

**Comment by koaning** [comment]
I'm now contemplating if it makes sense to introduce pelican for this. The idea is to *only* use pelican to generate the RSS. Worth exploring as an option mainly because pelican has great support.

**Comment by koaning** [comment]
I just learned about [kill the newsletter](https://kill-the-newsletter.com/). The idea is that it can transform any newsletter into an RSS feed. I'm digging it, but I figured I should if there are downsides to this. 

One downside that I can imagine is that I might want to switch email providers which would mean that we'd need to reset the newsletter. This is pretty much the only downside that I could come up with though.
